rocketpy/motors/tank.py

```diff
@@ -410,6 +409,73 @@ def inertia(self):
         """
         return self.liquid_inertia + self.gas_inertia
 
+    def _check_volume_bounds(self):
+        """Checks if the tank is overfilled or underfilled. Raises a ValueError
+        if the tank is overfilled or underfilled.
+        """
+        if (self.fluid_volume > self.geometry.total_volume + 1e-6).any():
+            raise ValueError(
+                f"The tank '{self.name}' is overfilled. The fluid volume is "
+                + "greater than the total volume of the tank.\n\t\t"
+                + "Try increasing the tank height and check out the fluid density"
+                + " values.\n\t\t"
+                + f"The fluid volume is {np.max(self.fluid_volume.y_array):.3f} m³ at "
+                + f"{self.fluid_volume.x_array[np.argmax(self.fluid_volume.y_array)]:.3f} s."
+            )
```

Comment:
```suggestion
            raise ValueError(
                f"The tank '{self.name}' is overfilled. The fluid volume is "
                + "greater than the total volume of the tank.\n\t\t"
                + "Try increasing the tank height and check out the fluid density"
                + " values.\n\t\t"
                + f"The fluid volume is {np.max(self.fluid_volume.y_array):.3f} m³ at "
                + f"{self.fluid_volume.x_array[np.argmax(self.fluid_volume.y_array)]:.3f} s.\n\t\t"
                + f"The tank total volume is {self.geometry.total_volume:.3f} m³."
            )
```

Reply:
I suggest that we also print the tank total volume all the over/underfilled error messages